### ISSUE TITLE
Reading worksheets causes stuck problems

### ISSUE THREAD

**Issue by robincodex** [EPPlusSoftware/EPPlus#1440]
### EPPlus usage

Personal use

### Environment

Windows

### Epplus version

7.1.2

### Spreadsheet application

Microsoft365 Excel

### Description

This [test.xlsx](https://github.com/EPPlusSoftware/EPPlus/files/15258886/default.xlsx) is my export from [Tencent Cloud Document](https://docs.qq.com/home), when I read the worksheets appeared to be stuck, I determined through debug appeared in the `book.Workbook.Worksheets`, I'm not sure whether it is a file format problem or EPPlus parsing problem, I use other xlsx parsing tools, such as [MiniExcel](https://github.com/mini-software/MiniExcel) and [excelize](https://github.com/qax-os/excelize) all parse fine.

```csharp
var book = new ExcelPackage();
await book.LoadAsync(FilePath);
var sheets = book.Workbook.Worksheets;
```

**Comment by JanKallman** [comment]
Thanks for reporting this. I will provide a fix shortly.

**Comment by JanKallman** [comment]
Fixed in EPPlus 7.1.3
